README.md

```diff
@@ -392,7 +392,7 @@ instance = SingleVMOpenEdXInstance.objects.create(
     sub_domain='dogwood.sandbox',
     name='Dogwood',
     fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+    branch_name='named-release/dogwood.rc',
```

Comment:
@itsjeyd That would work for me to leave it alone on this PR, yep.

I don't see an update to the README in #63 though - so it will be even more out of date after #63 is merged?